functions/api.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express'
+import { ApolloServer } from 'apollo-server-cloud-functions'
+import { buildSchema } from 'type-graphql'
+import { HelloWorldResolver } from './GraphQL/resolvers'
```

Comment:
`GraphQL` is an awkward directory name here. Since this is part of the `api` function, it should be in a directory named `api`. You can then move `api.ts` to `api/index.ts` to represent the relationship there.